client.go

```diff
@@ -262,6 +268,10 @@ func New(config Config) *Client {
 		c.Config.PingDelay = 600 * time.Second
 	}
 
+	if c.Config.PingTimeout == 0 {
+		c.Config.PingTimeout = c.Config.PingDelay + (60 * time.Second)
```

Comment:
This logic should not include references to `c.Config.PingDelay`. `c.Config.PingTimeout` should just be the timeout value.